### PR TITLE
feat(pack): add elixir/phoenix pack

### PR DESCRIPTION
**Pull request by treshenry**
## 📑 Description

There's a fair amount of interest in using AstroNvim for Elixir/Phoenix projects, see: https://elixirforum.com/t/getting-heex-and-h-working-in-nvim-with-astronvim for a bit more context. It would be nice to have a single thing people can use that gives them everything they need. Most stuff works out of the box, there's just a tiny bit of missing config.

### CODE REVIEW

**Comment by Uzaaft** [comment]
@treshenry  How about https://github.com/elixir-tools/elixir-tools.nvim ? Would it also fit into the pack

**Comment by treshenry** [comment]
@Uzaaft Definitely up for debate. My goal was to keep this super minimal so anyone getting started with AstroNvim for a Phoenix project could just grab this and start coding. I use elixir-tools myself but it doesn't quite feel like a default answer yet. I noticed there are packs like `typescript-all-in-one` and `python-ruff` for an alternate lsp, so maybe one pack with and one without elixir-tools? I think @mehalter already hates me for putting up this PR though, so... 😛

**Comment by Uzaaft** [comment]
> @Uzaaft Definitely up for debate. My goal was to keep this super minimal so anyone getting started with AstroNvim for a Phoenix project could just grab this and start coding. I use elixir-tools myself but it doesn't quite feel like a default answer yet. I noticed there are packs like `typescript-all-in-one` and `python-ruff` for an alternate lsp, so maybe one pack with and one without elixir-tools? I think @mehalter already hates me for putting up this PR though, so... 😛

I mean, doesn't elixir-tools enable other stuff as well?

**Comment by Uzaaft** [comment]
Let's add this as is, and we can add the elixir tools pack in a later PR if anyone is interested enough to add it.

File: lua/astrocommunity/pack/elixir-phoenix/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Elixir and Phoenix Pack
+
+Adds baseline support for Elixir/Phoenix projects.
+
+It:
```
Comment (Uzaaft) [inline lua/astrocommunity/pack/elixir-phoenix/README.md:5]:
```suggestion
This plugin pack does the following:
```

**Review by Uzaaft — APPROVED** [review]

**Comment by treshenry** [comment]
@Uzaaft I'm happy to put up another PR for an elixir-tools option.

Anything preventing this one from getting merged?

**Comment by Uzaaft** [comment]
> @Uzaaft I'm happy to put up another PR for an elixir-tools option.
> 
> Anything preventing this one from getting merged?

I'll merge it now. I was just waiting on the last check last evening and fell asleep 😂

**Review by Uzaaft — APPROVED** [review]

**Pull request merged**